### PR TITLE
Add known issues section to vmware page

### PR DESCRIPTION
**Pull request by soapy1**

### CODE REVIEW

File: website/pages/vmware/index.jsx [hunk 1]
```diff
@@ -325,6 +325,29 @@ export default function VmwareIndex() {
...
+              <p className="g-type-body">
+                There are some notable issues for the VMWare provider on Big
+                Sur.
+                <ul>
+                  <li><a href="https://github.com/hashicorp/vagrant/issues/12045">Port forwarding</a></li>
+                  <li><a href="https://github.com/hashicorp/vagrant/issues/11682">Restoring snapshots</a></li>
+                  <li><a href="https://github.com/hashicorp/vagrant/issues/12052">Creating networking devices</a></li>
+                </ul>
+                Please also see the "Known Issues" section of the
+                <a href="https://docs.vmware.com/en/VMware-Fusion/12/rn/VMware-Fusion-12-Release-Notes.html"> VMWare 12 relese notes</a>.
```
Comment (pearkes) [inline website/pages/vmware/index.jsx:345]:
```suggestion
                 <a href="https://docs.vmware.com/en/VMware-Fusion/12/rn/VMware-Fusion-12-Release-Notes.html">VMWare 12 relese notes</a>.
```

File: website/pages/vmware/index.jsx [hunk 2]
```diff
@@ -325,6 +325,29 @@ export default function VmwareIndex() {
...
+
+      <section id="faq" className={s.faq}>
+        <div className="g-grid-container">
+          <div className={s.tag}>Issues</div>
+          <h2 className={s.h2}>Known Issues</h2>
+          <ul>
+            <li>
+              <h4>Big Sur</h4>
+              <p className="g-type-body">
+                There are some notable issues for the VMWare provider on Big
```
Comment (pearkes) [inline website/pages/vmware/index.jsx:337]:
```suggestion
                There are some notable issues for the VMWare provider on Mac OS Big
```

**Review by pearkes — APPROVED** [review]
Looks good to me. Some minor suggestions (I didn't actually run the web page in a browser, so if the whitespace one is wrong that's fine).

File: website/pages/vmware/index.jsx (hunk 1 above)
Comment (soapy1) [reply website/pages/vmware/index.jsx:345]: This change makes the spacing a little funky

Screen Shot 2021-01-12 at 11 56 26 AM

Comment (pearkes) [reply website/pages/vmware/index.jsx:345]: Ah yeah was worried about that, please ignore!

File: website/pages/vmware/index.jsx [hunk 3]
```diff
@@ -325,6 +325,28 @@ export default function VmwareIndex() {
...
+          <h2 className={s.h2}>Known Issues</h2>
+          <ul>
+            <li>
+              <h4>Big Sur</h4>
+              <p className="g-type-body">
+                There are some notable issues for the VMWare provider on Mac OS Big
+                Sur. For more information please refer to the known issues section
+                in the documentation for Big Sur:
+                <ul>
+                  <li><a href="https://www.vagrantup.com/docs/known-issues#big-sur-related-issues">Big Sur Known Issues</a></li>
```
Comment (soapy1) [inline website/pages/vmware/index.jsx:341]:
```suggestion
                  <li><a href="https://www.vagrantup.com/docs/providers/vmware/known-issues">Big Sur Known Issues</a></li>
```

**Review by chrisroberts — APPROVED** [review]

**Pull request merged**